src/core/init/initialize_media_source.ts

```diff
@@ -107,6 +107,8 @@ export interface IInitializeArguments {
     maxBufferBehind$ : Observable<number>;
     /** Strategy when switching the current bitrate manually (smooth vs reload). */
     manualBitrateSwitchingMode : "seamless" | "direct";
+    /** Stategy when switching of audio track (smooth vs flush vs reload). */
+    audioTrackSwitchingMode : string;
```

Comment:
Why not directly listing the possibilities in the type? (This might imply to check them in `src/api/option_parsers.ts`)

Reply:
In the new version, I pushed, I now take into consideration the different specific cases: `smooth` | `flush` | `reload`.